src/test/java/com/github/dariobalinzo/task/ElasticSourceTaskTest.java

```diff
@@ -364,8 +416,9 @@ public void shouldRunSourceTaskWithJsonCastFilter() throws IOException, Interrup
         //when (fetching first page)
         task.start(conf);
         List<SourceRecord> poll1 = task.poll();
-        assertEquals("Struct{fullName=\"Test\",nonavrofield=non-avro-field,avroField=avro-field,age=10,ts=111}", poll1.get(0).value().toString());
-
+        Struct structValue = (Struct) poll1.get(0).value();
+        assertEquals(structValue.get("avroField"), "avro-field");
+        assertEquals(structValue.get("nonavrofield"), "non-avro-field");
         task.stop();
```

Comment:
Similar here, the string fullname is being cast to json: so we need to check that is actual a string literal beginning and ending by `"` (`fullName=\"Test\"`)